application/positionmanager/src/positionmanager/main.py

```
@@ -1,13 +1,22 @@
-from fastapi import FastAPI, HTTPException
 import os
-from datetime import datetime, timedelta
+from datetime import UTC, datetime, timedelta
+from typing import Any
+
 import polars as pl
 from typing import Dict, Any
```

Ruff (0.11.9)

`typing.Dict` is deprecated, use `dict` instead

(UP035)